### ISSUE TITLE
sandwich not working with current beta builds of Tesseract?

### ISSUE THREAD

**Issue by RNCTX** [jbarlow83/OCRmyPDF#271]
```
$ tesseract -v
tesseract 4.0.0-beta.1-370-g8b64
 leptonica-1.76.0 libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.5.3) : libpng 1.6.34+apng : libtiff 4.0.9 : zlib 1.2.11 : libwebp 1.0.0 : libopenjp2 2.3.0
```
...

```
$ pip list
Package                Version  
---------------------- ---------
cffi                   1.11.5    
img2pdf                0.2.4  
ocrmypdf               6.2.0    
pdftotext              2.0.2    
Pillow                 5.1.0    
pip                    10.0.1   
PyPDF2                 1.26.0   
reportlab              3.4.0     
ruffus                 2.6.3    

```
...

```
$ unpaper --v
unpaper: option `--v' is ambiguous
unpaper 6.1
License GPLv2: GNU GPL version 2.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Usage: unpaper [options] <input-file(s)> <output-file(s)>
```

When trying to use the sandwich renderer, I get...

```
$ ocrmypdf --deskew --pdf-renderer sandwich --output-type pdfa --pdfa-image-compression jpeg CenturyIIB-origscan.pdf CenturyIIB-sandwich.pdf

ERROR - The 'sandwich' renderer requires Tesseract 3.05.01 or newer; or Tesseract 4.00 alpha newer than February 2017.
```

**Comment by jbarlow83** [comment]
It should work.

The actual test that is failing asks whether ``tesseract --print-parameters`` contains the line ``textonly_pdf`` which is the indicator that the required features are available.

**Comment by jbarlow83** [comment]
Looks like tesseract broke ``--print-parameters``

**Comment by RNCTX** [comment]
Thanks for the reply.  I agree, there seems to be nothing wrong with the eventual functionality, just failing a version check.

**Comment by jbarlow83** [comment]
fixed in v6.2.1
